### PR TITLE
Fum autoboot

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

File: UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c [hunk 1]
```diff
@@ -1111,6 +1128,17 @@ WarnIfFirmwareUpdateMode (
       NULL
       );
 
+  //
+  // Create volatile runtime variable so applications can detect FUM
+  //
+  Status = gRT->SetVariable (
+      L"FirmwareUpdateMode",
+      &gDasharoSystemFeaturesGuid,
+      EFI_VARIABLE_RUNTIME_ACCESS,
```
Comment (miczyg1) [inline UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c]: Is EFI_VARIABLE_BOOTSERVICE_ACCESS not needed? iPXE does not operate at Runtime and VariableServiceQueryVariableInfo seems to require both BS and RT if RT is set

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c (hunk 1 above)
Comment (miczyg1) [reply UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c]: Yes, it is needed. Page 215 of [UEFI Specification 2.10](https://uefi.org/sites/default/files/resources/UEFI_Spec_2_10_Aug29.pdf):

```
• Runtime access to a data variable implies boot service access. Attributes that have
EFI_VARIABLE_RUNTIME_ACCESS set must also have EFI_VARIABLE_BOOTSERVICE_ACCESS
set. The caller is responsible for following this rule.
```

Comment (krystian-hebel) [reply UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c]: Fixed in https://github.com/Dasharo/edk2/compare/c4af5bac31238439de19e432e57eb3d0c5481f38..813a25418b2c3b636ea4e25765a705d3b3d29498

Comment (miczyg1) [reply UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c]: It is sufficient to link just the commit hash addressing the review (if it is just a single commit)

**Comment by miczyg1** [comment]
@krystian-hebel we can drop the commit with submodule bump, because I just merged different PR which bumper it and it already contains your changes

**Comment by krystian-hebel** [comment]
I've pushed rebased code but for some reason GUI doesn't see it...

EDIT: it does now, needed more time than usual.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**